app/controllers/allocations_controller.rb

```diff
@@ -27,11 +27,15 @@ def create
     group = Group.find(allocation_params[:group_id])
     user = User.find(allocation_params[:user_id])
     amount = allocation_params[:amount]
+    notify = allocation_params[:notify]
+    params[:allocation].delete(:notify)
```

Comment:
the notify data is getting removed bc we don't want to save it to the db. if i don't remove it here, it errors out when i try to save `allocation` bc there is no `notify` column on the table.